client/app/main/pdf-upload/pdf-upload.controller.js

```diff
@@ -2,87 +2,93 @@
 
 var app = angular.module('app');
 
-app.controller('PDFUploadCtrl',
-  function($scope, AbsenceRecord, Auth, School, Upload, toastr) {
-    $scope.forms = {};
-    $scope.isUploaded = false;
+function PDFUploadCtrl($scope, AbsenceRecord, Auth, School, Upload, toastr) {
+  $scope.maxDate = Date.now();
 
-    if (Auth.getCurrentUser().role === 'teacher') {
-      $scope.defaultSchool = Auth.getCurrentUser().assignment;
-      $scope.schools = [$scope.defaultSchool];
-    } else {
-      $scope.schools = School.query();
-    }
+  $scope.forms = {};
+  $scope.date = Date.now();
+  $scope.isUploaded = false;
 
-    $scope.data = {upload: {school: $scope.defaultSchool}};
+  if (Auth.getCurrentUser().role === 'teacher') {
+    $scope.defaultSchool = Auth.getCurrentUser().assignment;
+    $scope.schools = [$scope.defaultSchool];
+  } else {
+    $scope.schools = School.query();
+  }
 
-    $scope.upload = function(file) {
-      return Upload.upload({
-        url: '/api/pdfs/',
-        data: { file: file,
-                schoolId: $scope.data.upload.school._id,
-                date: $scope.date
-              }
-      });
-    };
+  $scope.data = {upload: {school: $scope.defaultSchool}};
 
-    $scope.cancelUpload = function() {
-      $scope.isUploaded = false;
-      $scope.result.data = {};
-      $scope.data.upload.message = 'Upload Cancelled';
-    };
+  $scope.upload = function(file) {
+    return Upload.upload({
+      url: '/api/pdfs/',
+      data: {
+        file: file,
+        schoolId: $scope.data.upload.school._id,
+        date: $scope.date
+      }
+    });
+  };
 
-    $scope.confirmUpload = function() {
-      AbsenceRecord.save({}, $scope.result.data, function(res) {
-        $scope.data.upload.message = 'Upload Confirmed!';
-        $scope.result.data = {};
-        $scope.isUploaded = false;
+  $scope.cancelUpload = function() {
+    $scope.isUploaded = false;
+    $scope.result.data = {};
+    $scope.data.upload.message = 'Upload Cancelled';
+  };
 
-        var schoolName = res.record.school.name;
-        if (res.students.length) {
-          toastr.success(
-            res.students.length + ' new students added.',
-            schoolName,
-            {timeOut: 10000}
-          );
-        }
+  $scope.confirmUpload = function() {
+    AbsenceRecord.save({}, $scope.result.data, function(res) {
+      $scope.data.upload.message = 'Upload Confirmed!';
+      $scope.result.data = {};
+      $scope.date = Date.now();
```

Comment:
I noticed in earlier versions of the app, the date was not reset. I'm assuming that this will reset the date to current date after confirmation of upload?